src/app/page.tsx

```diff
@@ -68,7 +69,10 @@ export default function Page() {
                   asChild
                 >
                   <a href={social.url}>
-                    <social.icon className="h-4 w-4" />
+                    {React.createElement(
+                      social.icon as React.ComponentType<{ className: string }>,
+                      { className: "h-4 w-4" },
+                    )}
```

Comment:
what's the reason for this change? what was the error?

Reply:
This is only if we're adding types. 
I'll remove this if types aren't necessary.